.storybook/preview.tsx

```diff
@@ -1,11 +1,11 @@
-import { Stats } from '@react-three/drei';
+import { Stats } from 'glodrei';
```

Comment:
Is this the right package?

Reply:
@amcdnl 

https://github.com/pmndrs/drei/pull/1819 

In order to produce a temporary solution until the pr is approved, we changed the package name and threw an update. 

@zekiyasin  
we are working on the same project and we had to come up with an urgent solution.

Reply:
OK - I'm OK with this but can you make a issue to remind us to come back and update to official?

Reply:
we opened a PR for @react-three/drei when they accept PR we can back to them